copycat/api/src/main/java/org/apache/kafka/copycat/connector/CopycatRecord.java

```diff
@@ -57,10 +62,18 @@ public Object getKey() {
         return key;
     }
 
+    public Schema getKeySchema() {
```

Comment:
We generally don't do getters/setters as a naming convention (since then people add them willy nilly). So  keySchema(), value(), etc. would be more convenient with the rest of the code.